frontend/packages/knative-plugin/console-extensions.json

```
@@ -409,6 +409,67 @@
       "required": ["KNATIVE_SERVING"]
     }
   },
+  {
+    "type": "console.page/route",
+    "properties": {
+      "exact": true,
+      "path": ["/functions/all-namespaces", "/functions/ns/:ns"],
+      "component": {
+        "$codeRef": "overviewComponent.FunctionsPage"
+      }
+    },
+    "flags": {
+      "required": ["KNATIVE_SERVING"]
+    }
+  },
+  {
+    "type": "console.page/route",
+    "properties": {
+      "exact": true,
+      "path": ["/functions"],
+      "component": {
+        "$codeRef": "knativeComponent.NamespaceRedirect"
+      }
+    },
+    "flags": {
+      "required": ["KNATIVE_SERVING"]
+    }
+  },
+  {
+    "type": "console.navigation/href",
+    "properties": {
+      "id": "functions",
+      "perspective": "dev",
+      "section": "resources",
+      "name": "%knative-plugin~Functions%",
+      "href": "/functions",
+      "namespaced": true,
+      "insertBefore": "builds",
+      "dataAttributes": {
+        "data-quickstart-id": "qs-nav-functions",
+        "data-test-id": "functions-details-header"
+      }
+    },
+    "flags": {
+      "required": ["OPENSHIFT", "KNATIVE_SERVING", "KNATIVE_SERVING_SERVICE"]
+    }
+  },
+  {
+    "type": "console.page/route",
+    "properties": {
+      "exact": false,
+      "path": ["/functions/ns/:ns/:name"],
+      "model": {
+        "group": "serving.knative.dev",
+        "version": "v1",
+        "kind": "Service"
+      },
+      "component": { "$codeRef": "overviewComponent.FunctionDetailsPage" }
+    },
+    "flags": {
+      "required": ["OPENSHIFT", "KNATIVE_SERVING", "KNATIVE_SERVING_SERVICE"]
```

Same as above. It's a feature that works on all clusters with Knative. Please remove OPENSHIFT from the dependency list:

```suggestion
      "required": ["KNATIVE_SERVING", "KNATIVE_SERVING_SERVICE"]
```